src/ui/public/agg_types/metrics/latest.js

```diff
@@ -0,0 +1,42 @@
+import AggTypesMetricsMetricAggTypeProvider from 'ui/agg_types/metrics/metric_agg_type';
+import latestEditor from 'ui/agg_types/controls/latest.html';
+
+export default function AggTypeMetricAvgProvider(Private) {
+  let MetricAggType = Private(AggTypesMetricsMetricAggTypeProvider);
+
+  return new MetricAggType({
+    name: 'top_hits',
+    title: 'Latest',
+    makeLabel: function (aggConfig) {
+      return 'Latest ' + aggConfig.params.field.displayName;
```

Comment:
Can you make this `this.params.customLabel || 'Latest ' + aggConfig.params.field.displayName` in order to enable the custom label via the legend?